README.md

```diff
@@ -1,4 +1,17 @@
-# Customer.io Python bindings [![CircleCI](https://circleci.com/gh/customerio/customerio-python.svg?style=svg)](https://circleci.com/gh/customerio/customerio-python)
+<p align="center">
+  <a href="https://customer.io">
+    <img src="https://user-images.githubusercontent.com/6409227/144680509-907ee093-d7ad-4a9c-b0a5-f640eeb060cd.png" height="60">
+  </a>
+  <p align="center">Power automated communication that people like to receive.</p>
+</p>
+
+![PyPI](https://img.shields.io/pypi/v/customerio)
+![Software License](https://img.shields.io/github/license/customerio/customerio-python)
+[![Build status](https://github.com/customerio/customerio-python/actions/workflows/main.yml/badge.svg)](https://github.com/customerio/customerio-python/actions/workflows/main.yml)
+![PyPI - Python Version](https://img.shields.io/pypi/pyversions/customerio)
+![PyPI - Downloads](https://img.shields.io/pypi/dm/customerio)
+
+# Customer.io Python 
```

Comment:
Did you intentionally drop the word "bindings" here?

Reply:
Yeah, it was an outlier in that none of the other libraries identify themselves as "[language] bindings". So I've opted to standardize on everything being "Customer.io [language]" and avoided calling them anything more like "Customer.io [language] client" or "Customer.io [language] bindings".

I haven't merged the similar PRs to the [Node](https://github.com/customerio/customerio-node/pull/90), and [Go](https://github.com/customerio/go-customerio/pull/31) libraries yet (and won't until Monday at the earliest) so if there's a strong desire to call these libraries bindings that's fine by me. I care less about the syntax and more about consistency.